docker-compose.yml

```diff
@@ -79,6 +79,14 @@ services:
       - testing/browserstack_ci.env
     depends_on: []
 
+  ruby-tests:
```

Comment:
could we call this `ruby-style-tests` So we know it's not the physical ruby code being ran, but just checking the style.

Reply:
I considered that, but I figured that if there's any rspec or similar tests going in here in the future, this would clearly be the container to run them in.

Reply:
Hmmm oki. I don't think there would be, but seems sensible.